spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/DataSet.scala

```diff
@@ -548,6 +551,32 @@ object DataSet {
       rawData.coalesce(num, true)
     }
 
+    /**
+     * Extract hadoop sequence files from an HDFS path as ImageFrame
+     * @param url sequence files folder path
+     * @param sc spark context
+     * @param classNum class number of data
+     * @param partitionNum partition number, default: Engine.nodeNumber() * Engine.coreNumber()
+     * @return
+     */
+    private[bigdl] def filesToImageFrame(url: String, sc: SparkContext,
+      classNum: Int, partitionNum: Option[Int] = None): ImageFrame = {
+      val num = partitionNum.getOrElse(Engine.nodeNumber() * Engine.coreNumber())
+      val rawData = sc.sequenceFile(url, classOf[Text], classOf[Text], num).map(image => {
```

Comment:
suggested to change the value type to ByteWritable. Also change the value type in BGRImgToLocalSeqFile to ByteWritable. Load Text value in python seq file API would get encoding error if the true value type is byte although we only load seq file in scala.